airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -1,18 +1,12 @@
-import { setIn } from "formik";
+import { pipe } from "fp-ts/function";
```

Comment:
Any alternative ways of doing the same thing without having to add a new node module?

Reply:
Yeap, sure, we can use inline style instead of pipe
Exmaple:
//  Inline
`const finalValue = doFinalSthg(doSthgElse(addSthg(value)));`

// With pipe
`const finalValue = pipe(value, addSthg, doSthgElse, doFinalSthg);`

But I'm personally prefer using pipe since it increase the code readability.
I've added the `fp-ts` not for using the `pipe` only.